lib/decompress/zstd_decompress_block.c

```diff
@@ -974,11 +974,20 @@ ZSTD_decompressSequences_body( ZSTD_DCtx* dctx,
                 BIT_DStream_endOfBuffer < BIT_DStream_completed &&
                 BIT_DStream_completed < BIT_DStream_overflow);
 
+#ifdef __aarch64__
+        BIT_reloadUnfinishedDStream(&(seqState.DStream));
```

Comment:
Same here, this shouldn’t impact performance significantly

Reply:
> What gains come from member variable reordering?
> 
> We may want to split each individual optimization into its own PR so that we can selectively merge them as they’re ready. That will probably be the easiest way to work when both sides need to do measurements.

Ok, I will split them,  and measure their gains.